xl/metadata/_id3.py

```diff
@@ -105,6 +107,11 @@ def _get_tag(self, raw, t):
         elif t == 'COMM':  # Newlines within comments are allowed, keep them
             for item in field:
                 ret.extend([value for value in item.text])
+        elif t == 'POPM':
+            # Rating Stars
+            data = field[0].rating
+            rating = self._rating_to_stars(data)
```

Comment:
What about tags with multiple POPM fields (each with a unique email?) I think that's how it's supposed to work.

Reply:
I think this would work on mp3 only. flac/vorbis and mp4 only use one rating for all.